### PR TITLE
app: set new notes by wiping old ones first

### PR DESCRIPTION
**Pull request by chappjc**
Resolves https://github.com/decred/dcrdex/issues/758

### CODE REVIEW

File: client/db/bolt/db.go [hunk 1]
```diff
@@ -12,7 +12,6 @@ import (
 	"sort"
 	"time"
 
-	"decred.org/dcrdex/client/db"
 	dexdb "decred.org/dcrdex/client/db"
```
Comment (buck54321) [inline client/db/bolt/db.go:15]: Didn't even know this would compile like this.

**Review by buck54321 — COMMENTED** [review]
I think this is a javascript bug you could correct real quick. 

Replace the commented line at https://github.com/decred/dcrdex/blob/3a26fcbf5b154ff87f20335e9ff5485a7e213ac3/client/webserver/site/src/js/app.js#L445

with

```javascript
this.notes = []
```

I did a few quick tests and it seems to correct the issue.

**Comment by chappjc** [comment]
> ```js
> this.notes = []
> ```

Ah, so this wipes out old notes from local storage from previous runs (rather than adding to them)?

**Comment by buck54321** [comment]
Just from memory here, but outdated the browser storage will be overwritten in `prependNoteElement` too.

**Comment by chappjc** [comment]
Hmm.  I've noticed that when I nuke dex.db and start fresh, I sometimes end up with ancient notes from before, and these were causing the harmless errors in https://github.com/decred/dcrdex/issues/758.  Not always though.  I think not acking them before nuking is the key to repro.

**Comment by buck54321** [comment]
That's right. Before, the ancient notes were loaded into `this.notes` on startup. When `app.setNotes` is called from `RegisterPage` or `LoginPage`, new notes were appended to the ancient notes in `this.notes`. Only the new ones would be displayed in the bell. Then, when you clicked the bell, we were trying to ack all of the unacked notes in `this.notes`, which still contained the ancients.

File: client/db/bolt/db.go (hunk 1 above)
Comment (chappjc) [reply client/db/bolt/db.go]: removed all the go changes

**Comment by chappjc** [comment]
This is now included https://github.com/decred/dcrdex/pull/781, but merging anyway.

**Pull request merged**